html/constants.js

```diff
@@ -0,0 +1,143 @@
+export const calories_per_day = 75,
+	seg_time = 30,
+	total_day_time = seg_time * 16,
+	day_segs = 10,
+	dusk_segs = 4,
+	night_segs = 2,
+	day_time = seg_time * day_segs,
+	dusk_time = seg_time * dusk_segs,
+	night_time = seg_time * night_segs,
+
+	perish_warp = 1,
+
+	stack_size_largeitem = 10,
+	stack_size_meditem = 20,
+	stack_size_smallitem = 40,
+
+	healing_tiny = 1,
+	healing_small = 3,
+	healing_medsmall = 8,
+	healing_med = 20,
+	healing_medlarge = 30,
+	healing_large = 40,
+	healing_huge = 60,
+	healing_superhuge = 100,
+
+	sanity_supertiny = 1,
+	sanity_tiny = 5,
+	sanity_small = 10,
+	sanity_med = 15,
+	sanity_medlarge = 20,
+	sanity_large = 33,
+	sanity_huge = 50,
+
+	perish_one_day = 1 * total_day_time * perish_warp,
+	perish_two_day = 2 * total_day_time * perish_warp,
+	perish_superfast = 3 * total_day_time * perish_warp,
+	perish_fast = 6 * total_day_time * perish_warp,
+	perish_fastish = 8 * total_day_time * perish_warp,
+	perish_med = 10 * total_day_time * perish_warp,
+	perish_slow = 15 * total_day_time * perish_warp,
+	perish_preserved = 20 * total_day_time * perish_warp,
+	perish_superslow = 40 * total_day_time * perish_warp,
+
+	dry_superfast = 0.25 * total_day_time,
+	dry_veryfast = 0.5 * total_day_time,
+	dry_fast = total_day_time,
+	dry_med = 2 * total_day_time,
+
+	calories_tiny = calories_per_day / 8, // berries
+	calories_small = calories_per_day / 6, // veggies
+	calories_medsmall = calories_per_day / 4,
+	calories_med = calories_per_day / 3, // meat
+	calories_large = calories_per_day / 2, // cooked meat
+	calories_huge = calories_per_day, // crockpot foods?
+	calories_superhuge = calories_per_day * 2, // crockpot foods?
+
+	hot_food_bonus_temp = 40,
+	cold_food_bonus_temp = -40,
+	food_temp_brief = 5,
+	food_temp_average = 10,
+	food_temp_long = 15,
+
+	spoiled_health = -1,
+	spoiled_hunger = -10,
+	perish_fridge_mult = .5,
+	perish_ground_mult = 1.5,
+	perish_global_mult = 1,
+	perish_winter_mult = .75,
+	perish_summer_mult = 1.25,
+
+	stale_food_hunger = .667,
+	spoiled_food_hunger = .5,
+
+	stale_food_health = .333,
+	spoiled_food_health = 0,
+
+	base_cook_time = night_time * .3333,
+
+	defaultStatMultipliers = {
+		raw: 1,
+		dried: 1,
+		cooked: 1,
+		recipe: 1
+	},
+
+	VANILLA = 1,
+	GIANTS = 1 << 1,
+	SHIPWRECKED = 1 << 2,
+	TOGETHER = 1 << 3,
+	WARLY = 1 << 4,
+	HAMLET = 1 << 5,
+
+	modeMask = VANILLA | GIANTS | SHIPWRECKED | HAMLET,
```

Comment:
`modeMask` isn't a constant. I think it's working because browsers may not be respecting the `const` declaration with exports, or something along those lines. This can just be moved to a normal variable in the main `foodguide.js` file.

Edit: looks like this is also a variable in `foodguide.js`, so that explains why things are working more nicely. So this line can just be deleted.

Reply:
Ok, I didn't delve extensively in the page logic, so unfortunately I missed this.
```suggestion
```